staging/src/k8s.io/apiserver/pkg/admission/plugin/validatingadmissionpolicy/typechecking.go

```diff
@@ -72,7 +73,8 @@ func (c *TypeChecker) Check(policy *v1alpha1.ValidatingAdmissionPolicy) []v1alph
 	for _, v := range policy.Spec.Validations {
 		exps = append(exps, v.Expression)
 	}
-	msgs := c.CheckExpressions(exps, policy.Spec.ParamKind != nil, policy)
+	// TODO(jiahuif) hasAuthorizer always true for now, will change after expending type checking to all fields.
```

Comment:
Note: The type checking support for message expression is added in https://github.com/kubernetes/kubernetes/pull/119209. After that is merged, a followup PR will be required to address this TODO. Thank you!